### ISSUE TITLE
Add a css file

### ISSUE THREAD

**Issue by daniel-jozsef** [paulcollett/react-masonry-css#11]
So since this project is in npm, and we have really good support in webpack for css files, I'd like to see a css file in the distribution that I can just import, as opposed to having it in the documentation. ;)

**Comment by paulcollett** [comment]
Good idea! This is ideal and I'll look into it, though for simplicity of setup how would you feel about automatic css styling without needing to add css? Would be outputted as inline styles.

**Comment by lol-russo** [comment]
Inline styles would be fine with me, FWIW. If you're sticking with external CSS file then I'd love to see what is proposed above implemented :)
